stripper/ze_doom3_v2.cfg

```diff
@@ -1611,4 +1611,187 @@ modify:
 		"OnPressed" "strp_stg1_zm_elev_pushEnable20.51"
 		"OnPressed" "strp_stg1_zm_elev_pushKill321"
 	}
+}
+; Decrease potency of push item to reduce chances of accidental boosting
+modify:
+{
+	match:
+	{
+		"classname" "trigger_push"
+		"targetname" "pushgun_push"
+	}
+	replace:
+	{
+		"speed" "800"
+	}
+}
+; Alter npc vscript logic so target selection is random
+; ############################### THIS CHANGE WILL NOT WORK WITHOUT HAVING ################################
+; ############################## csgo/scripts/vscripts/gfl/boss_move_tar.nut ##############################
+; ########################################## IN THE SERVER FILES ##########################################
+; ######### https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/master/vscripts/boss_move_tar.nut #########
+modify:
+{
+	match:
+	{
+		"targetname" "Boss_Move_Physbox"
+		"vscripts" "doom3/boss_move_tar.nut"
+	}
+	replace:
+	{
+		"vscripts" "gfl/boss_move_tar.nut"
+	}
+}
+modify:
+{
+	match:
+	{
+		"targetname" "Boss_Move_Physbox1"
+		"vscripts" "doom3/boss_move_tar.nut"
+	}
+	replace:
+	{
+		"vscripts" "gfl/boss_move_tar.nut"
+	}
+}
+modify:
+{
+	match:
+	{
+		"targetname" "npc_physbox"
+		"vscripts" "doom3/boss_move_tar.nut"
+	}
+	replace:
+	{
+		"vscripts" "gfl/boss_move_tar.nut"
+	}
+}
+modify:
+{
+	match:
+	{
+		"targetname" "npc_physbox1"
+		"vscripts" "doom3/boss_move_tar.nut"
+	}
+	replace:
+	{
+		"vscripts" "gfl/boss_move_tar.nut"
+	}
+}
+; Add zm tp to stage 3 rotating boss fight. This will tp the cts that turn into zms that edge the zm pit. Will not tp the zms that enter the arena naturally
+add:
+{
+	"model" "*225"
+	"classname" "trigger_multiple"
+	"origin" "2378.56 -3680 7594"
+	"spawnflags" "1"
+	"StartDisabled" "0"
+	"filtername" "zm"
+	"OnStartTouch" "!activatorAddContextzm_boss_arena:10-1"
+}
+add:
+{
+	"origin" "2176 -3904 7498"
+	"classname" "filter_activator_context"
+	"targetname" "filter_zm_boss_arena"
+	"ResponseContext" "zm_boss_arena"
+	"Negated" "1"
+}
+add:
+{
+	"origin" "2144 -3936 7466"
+	"classname" "filter_multi"
+	"targetname" "filter_stg3_zm_tp"
+	"filtertype" "0"
+	"Negated" "0"
+	"Filter01" "zm"
+	"Filter02" "filter_zm_boss_arena"
+}
+add:
+{
+	"model" "*330"
+	"targetname" "strp_stg3_rot_boss_loop"
+	"classname" "trigger_multiple"
+	"origin" "-80.5 -3993.5 7993.12"
+	"spawnflags" "1"
+	"StartDisabled" "1"
+	"wait" "0.39"
+	"OnTrigger" "!selfDisable0.2-1"
+	"OnTrigger" "!selfEnable0.4-1"
+	"OnTrigger" "strp_stg3_zm_tp_rot_bossEnable0-1"
+	"OnTrigger" "strp_stg3_zm_tp_rot_bossDisable0.39-1"
+}
```

Comment:
This is unnecessarily messy, just use a logic_timer to keep toggling only strp_stg3_zm_tp_rot_boss's enabled state.